### PR TITLE
Issue #138: Fixing typos and removing all instances of '// endif'

### LINKED ISSUE #138: Fix all typos in the comments

### PR DESCRIPTION
**Pull request by chris-pearce**
This fixes #138.

Also removed all instances of `// endif` at the end of `@if` statements.

### CODE REVIEW

**Comment by stowball** [comment]
All of the code plurals were correct as `tag`s, not the newly changed `tag`'s.

An apostrophe denotes ownership or a contraction, not a plural.

File: utilities/_u-hide.scss [hunk 1]
```diff
@@ -86,7 +86,7 @@ $u-hide-js-is-off-class:                           no-js !default;
 
 
   /**
-   * Form `legend`s need different treatment.
+   * Form `legend`'s need different treatment.
```
Comment (kllevin) [inline utilities/_u-hide.scss]: This use of the apostrophe is ok, as the legend belongs to the form (and you should only have one legend).

**Comment by chris-pearce** [comment]
Doesn't every html element have a parent: the `html` element :bomb:

**Comment by kllevin** [comment]
I think in general you need to decide if you want to refer to elements singular or plural. 

For example when discussing the `table` element you would say something like the `table`'s `rows`, as multiple rows belong to a single table. 

However when you say: Turn styling on for inputs and `select`'s you are mixing plural and possessive. It should be "Turn styling on for inputs and `select`s". 

In the second example it sounds better to refer to the elements in plural form. I guess it depends on each use case.

**Comment by chris-pearce** [comment]
My comment was a joke.

The apostrophes are no more.

**Pull request merged**